f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_igce_estimate.xml

```diff
@@ -36,23 +36,26 @@
         </element>
         <element label="Unit Price" max_length="20" name="unit_price" type="currency"/>
         <element label="Unit Quantity" max_length="4000" name="unit_quantity" type="string"/>
+        <element choice="3" choice_field="can_access_package" choice_table="x_g_dis_atat_contacts" default="NO" label="Updated Description" max_length="40" name="updated_description" type="choice">
```

Comment:
Looks good.  Choice type, references what we've traditionally (and unintuitively) used for Yes/No options